### DISCUSSION TITLE
Guide/tutorial for Podman + Bash scripts for personalized persistent multisession containers, similar to Toolbx and Distrobox

### DISCUSSION THREAD

**Discussion by castedo** [containers/podman#24847] [Show and tell]
A guide with tutorials is available at https://cnest.readthedocs.io/ for using small Bash scripts to call podman directly for personalized persistent multisession containers. This functionality is similar to what is provided by Toolbx and Distrobox. The start of the guide goes into some of the reason you might want to use Toolbx/Distrobox vs script directly to podman.

This guide and the scripts provided are the result of simplifying and reducing my own scripts since 2020 as Podman has improved and added features.

**Comment by abitrolly** [comment]
How do you solve the problem that newly created users can not write to `podman` volumes? (https://github.com/containers/podman/discussions/24856)

**Reply by castedo** [reply]
I have added comments to #24856 describing the approach taken by the `create-cnest` script.
